src/app/charts/content/templates/ark-secret-conn.yaml

```diff
@@ -22,6 +22,22 @@
     {{- $data = set $data "group" ($groups | b64enc) }}
     {{- $data = set $data "url" ($content.url | b64enc) }}
     {{- $data = set $data "ui" ($content.ui | b64enc) }}
+
+    {{- if (eq $content.dialect "box") }}
+      {{- $license := (include "arkcase.license" (dict "ctx" $ "name" "box") | fromYaml) }}
+      {{- if and $license $license.data -}}
+        {{- $license = $license.data -}}
+
+        {{- if not (kindIs "string" $license) -}}
+          {{- fail (printf "The Box RSA key in global.licenses.box must be a single string, not a %s" (kindOf $license)) -}}
+        {{- end }}
+
+        {{- $data = set $data "private-key" $license | default "" }}
```

Comment:
For consideration: you may want to decode (b64) the license string and perform some basic validation of the resulting value (not obligatory):

- is it a properly formatted string?
- does it _appear_ to be a correct license?

Etc.

Like I said: not obligatory, but may be helpful if we can do it easily to avoid rendering something that isn't going to work properly. If it's a lot of work and not something that can be done in a few lines of code, then it's probably not worth doing.